### DISCUSSION TITLE
Open file picker in cwd

### DISCUSSION THREAD

**Discussion by georgealexanderday** [zellij-org/zellij#3285] [Q&A] [answered]
Hi there,

How can I open the file picker in the cwd? Is there an argument I can pass to it? 

I've tried using the `-a` in combination with `$(pwd)`  but must be getting the format wrong here.

Using `zellij pipe -p filepicker` always puts me in system root `/` directory.

Any ideas?

**Comment by cristiand391** [comment, marked as answer]
Doesn't seem to be documented but the plugin takes a `caller_cwd` config value that allows that:

```
zellij pipe -p filepicker -c "caller_cwd=<path>"
```

https://github.com/zellij-org/zellij/blob/162a6dd93dc6c4d4d9a7eb1a7765623bc117e8f4/default-plugins/strider/src/main.rs#L41

**EDIT:**

Aram added this note about cwd in plugins:
https://zellij.dev/documentation/plugin-aliases.html?highlight=caller#a-note-about-cwd

**Reply by georgealexanderday** [reply]
Perfect - thank you!
